Coursework.java

```diff
@@ -0,0 +1,524 @@
+package CourseworkJan2020;
+

```

Comment:
Write a comment on a particular line of code
```suggestion
put this thing in... bla bla `bla`
```then `inset some `code`` @PaulLynch123 #1 iSsue nUmber oNe